src/CarbonAware.Aggregators/src/CarbonAware/CarbonAwareAggregator.cs

```diff
@@ -74,6 +75,22 @@ public async Task<IEnumerable<EmissionsForecast>> GetCurrentForecastDataAsync(ID
         }
     }
 
+    /// <inheritdoc />
+    public async Task<double> CalculateAverageCarbonIntensityAsync(IDictionary props)
+    {
+        using (var activity = Activity.StartActivity())
+        {
+            var start = GetOffsetOrDefault(props, CarbonAwareConstants.Start, DateTimeOffset.Now.ToUniversalTime());
+            var end = GetOffsetOrDefault(props, CarbonAwareConstants.End, start);
+            _logger.LogInformation("Aggregator getting average carbon intensity from data source");
+            var emissionData = await this._dataSource.GetCarbonIntensityAsync(GetLocationOrThrow(props), start, end);
+            var value = emissionData.AverageOverPeriod(start, end);
+            _logger.LogInformation($"Carbon Intensity Average: {value}");
```

Comment:
Any reason why this is LogInfo and we also have the same thing as a LogDebug [here in the controller](https://github.com/microsoft/carbon-aware-sdk/pull/111/files#diff-1f2e99bbca80e13b4196ed490313cd965b101c63cd15413f404da07b0a1e97deR245)

Can you speak to the use-case for these LogInfos?

Reply:
Well, yes, it was since both (a) the _SciScoreController_ version of _CalculateAverageCarbonIntensityAsync_ had the same line 88 above, and (b) the other methods in _CarbonAwareAggregator_ used _LogInformation_, so we kept it similar.